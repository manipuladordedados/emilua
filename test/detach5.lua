-- This test ensures that detached fibers who error will have their stacktrace
-- printed to stderr

local println = require('println')
local sleep_for = require('sleep_for')

local fib = spawn(function()
    println('secondary fiber starts')
    error('tag', 0)
end)

println('secondary fiber spawned')
sleep_for(10)
fib:detach()
println('end of main fiber')
