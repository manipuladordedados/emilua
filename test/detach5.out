secondary fiber spawned
secondary fiber starts
Fiber 0x0 panicked: 'tag'
stack traceback:
	input.lua:9: in function <input.lua:7>
end of main fiber
